app/views/my_facilities/drug_stocks/_drug_stocks_table.html.erb

```diff
@@ -3,9 +3,12 @@
     <colgroup>
```

Comment:
Praveen mentioned that the layout breaks if there's too many categories, can we check if it's addressed? https://simpledotorg.slack.com/archives/C03URFX2LUT/p1666007119061689?thread_ts=1665984305.235149&cid=C03URFX2LUT

Reply:
It's sort-of addressed @prabhanshuguptagit ... on narrow screens or if you add too many drug/dosage combinations to track, it still runs over the side of the card. This is not a blocker.